routers/api/v1/misc/signing.go

```diff
@@ -50,11 +51,70 @@ func SigningKey(ctx *context.APIContext) {
 		path = ctx.Repo.Repository.RepoPath()
 	}
 
-	content, err := asymkey_service.PublicSigningKey(ctx, path)
+	content, format, err := asymkey_service.PublicSigningKey(ctx, path)
 	if err != nil {
 		ctx.APIErrorInternal(err)
 		return
 	}
+	if format == "ssh" {
```

Comment:
```suggestion
	if format != "gpg" {
```

Reply:
I forget to mentioned this value is coming from git cli, gpg is not a valid value for git.

https://git-scm.com/docs/git-config#Documentation/git-config.txt-gpgformat

Based on the documentation we might preferable use the following instead
```suggestion
	if format != "" && format != "openpgp" {
```

Accepting an empty key format for gpg looked for me to be less likely to break anything I missed.

Reply:
Added constants for openpgp in the git module for the time being with a link to the git documentation. Only a single "openpgp" is now hardcoded for default settings.